.gitignore

```diff
@@ -21,4 +21,6 @@ csslint.out.xml
 *.orig
 *.tmp
 *.log
-_site/*
\ No newline at end of file
+_site/*
+
+.settings/*
```

Comment:
Why are you adding `.settings/*` here? Is it because you have local settings? You could also just omit them when you do a **git add** for a commit.